types/intercom-client/index.d.ts

```diff
@@ -4,8 +4,9 @@
 //                 Josef Hornych <https://github.com/peping>
 //                 Mikhail Monchak <https://github.com/mikhail-monchak>
 //                 Chris Doe <https://github.com/cdoe>
+//                 Daniel Cummings <https://github.com/dan-cummings>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 3.7.5
```

Comment:
This made your CI fail. See table with versions window:
https://github.com/DefinitelyTyped/DefinitelyTyped#older-versions-of-typescript-27-and-earlier
```suggestion
// TypeScript Version: 3.7
```